src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -22,7 +22,7 @@ public static void Main(string[] args)
 
         public static IWebHost BuildWebHost(string[] args)
         {
-            return CreateWebHostBuilder(args).Build();
+            return CreateWebHostBuilder(args).UseIIS().Build();
```

Comment:
This should be conditional on the environment. When running locally, we don't need this set.

Reply:
As discussed we leave it as is.

Reply:
👍 for others, verified and this is a noop when not in proc